templates/partial/_navigation.html

```diff
@@ -2,8 +2,8 @@
 
 <header id="navigation" class="p-navigation {%if not request.path|get_nav_path == 'partners'%}is-sticky{% endif %} is-dark">
```

Comment:
`Products` will need to be deleted it looks like

Reply:
so delete this, correct?
```
        <li class="p-navigation__item" id="products-nav">
          <a href="/#products" class="p-navigation__link">Products</a>
        </li>
```

Reply:
how do you go t the projects page we built then - @juanruitina ?